data/na/swagger/1-0-5.yaml

```diff
@@ -1581,7 +1581,9 @@ definitions:
         format:  double
         description: 'The commission amount collected against the fee. This field is only returned for channel partner accounts.'
   MerchantReport:
-    $ref: '#/definitions/MerchantRecord'
+    properties:
+      MerchantRecord:
```

Comment:
Other properties in this file are lower_snake_case.
Should this be merchant_record?

Reply:
The structure was fixed and now accomplish with the standard